.gitignore

```diff
@@ -56,3 +56,7 @@ doc/manual/build
 doc/manual/source/_build
 
 run/results.js
+
+Make.mach.claire-linux
```

Comment:
Are these .gitignore changes needed?

Reply:
Whoops, that snuck in there. I had it on the main of my fork and I'll pull it out in a later PR.